### PR TITLE
Added Tailwind CSS config icon

### PR DESCRIPTION
**Pull request by aparajita**
#### Description

I added a file icon for Tailwind CSS config files.

#### Motivation and Context

This file association did not exist. And because Tailwind rules. 😁 

#### How Has This Been Tested?

It hasn't been. The format of the icon_associations.xml seems pretty straightforward.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: src/main/resources/icon_associations.xml [hunk 1]
```diff
@@ -323,6 +323,7 @@
         <regex name="Swift" pattern=".*\.swift$" icon="/icons/files/swift.svg"/>
         <regex name="Swig" pattern=".*\.swig$" icon="/icons/files/swig.svg"/>
         <regex name="Symfony" pattern=".*\.symfony" icon="/icons/files/symfony.svg"/>
+        <regex name="Tailwind CSS Config" pattern="^tailwind(\..+)?\.js$" icon="/icons/files/tailwindcss.svg"/>
```
Comment (mallowigi) [inline src/main/resources/icon_associations.xml]: is it always js? no json? .tailwindrc or something?

Comment (mallowigi) [reply src/main/resources/icon_associations.xml]: or ts?

Comment (aparajita) [reply src/main/resources/icon_associations.xml]: No json. ts maybe sometime in the future. I adjusted the regex accordingly.

**Comment by aparajita** [comment]
Thank you!

**Pull request merged**